src/tests/ftest/erasurecode/aggregation.py

```diff
@@ -93,8 +93,8 @@ def test_ec_aggregation_time(self):
         # Set time mode aggregation
         self.pool.set_property("reclaim", "time")
         self.pool.connect()
-        print("pool_percentage Before = {} ".format(
-            self.pool.pool_percentage_used()))
+        self.log.info("pool_percentage Before = {} ".format(
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions